crates/uv/src/logging.rs

```diff
@@ -160,40 +166,38 @@ pub(crate) fn setup_logging(
         };
     let writer = std::sync::Mutex::new(anstream::AutoStream::new(std::io::stderr(), color_choice));
 
-    match level {
-        Level::Default | Level::Verbose => {
-            // Regardless of the tracing level, show messages without any adornment.
-            let format = UvFormat {
-                display_timestamp: false,
-                display_level: true,
-                show_spans: false,
-            };
-
-            tracing_subscriber::registry()
-                .with(durations_layer)
-                .with(
-                    tracing_subscriber::fmt::layer()
-                        .event_format(format)
-                        .with_writer(writer)
-                        .with_ansi(ansi)
-                        .with_filter(filter),
-                )
-                .init();
-        }
-        Level::ExtraVerbose => {
-            // Regardless of the tracing level, include the uptime and target for each message.
-            tracing_subscriber::registry()
-                .with(durations_layer)
-                .with(
-                    HierarchicalLayer::default()
-                        .with_targets(true)
-                        .with_timer(Uptime::default())
-                        .with_writer(writer)
-                        .with_ansi(ansi)
-                        .with_filter(filter),
-                )
-                .init();
-        }
+    let detailed_logging = std::env::var(EnvVars::UV_LOG_CONTEXT).is_ok();
+    if detailed_logging {
+        // Regardless of the tracing level, include the uptime and target for each message.
+        tracing_subscriber::registry()
+            .with(durations_layer)
+            .with(
+                HierarchicalLayer::default()
+                    .with_targets(true)
+                    .with_timer(Uptime::default())
+                    .with_writer(writer)
+                    .with_ansi(ansi)
+                    .with_filter(filter),
+            )
+            .init();
+    } else {
+        // Regardless of the tracing level, show messages without any adornment.
+        let format = UvFormat {
+            display_timestamp: false,
```

Comment:
Sometimes, I sometimes find timestamps very helpful (it's direct performance information), but it's probably not worth adding a flag for it.